added advice page and a new css file linked to index.html

Zosimi opened the pull request:
made changes to practise with git and github

acodeinprogress commented:
Hey @Zosimi,

This looks great so far! There are just a couple changes that will need to be made before we can merge this in:
1. We would like for your advice file to link to your personal CSS file rather than a style.css file.
2. It looks like you changed the index.html file to link to your personal CSS file. Instead, we'll want to remove that and in the `<body>`, add a link that directs to your advice page.

Once you do that, you can add, commit, and push and your changes will join your pull request here.

xoxo,
Sharon

Zosimi commented:
Hi Sharon,

Thank you for your feedback. I removed the css link from the index file and
added it to my personal advice page instead. I've sent a new pull request.

Thanks again,
Sandra

On Sun, 21 Apr 2019 at 15:02, Sharon <notifications@github.com> wrote:

> Hey @Zosimi <https://github.com/Zosimi>,
>
> This looks great so far! There are just a couple changes that will need to
> be made before we can merge this in:
>
>    1. We would like for your advice file to link to your personal CSS
>    file rather than a style.css file.
>    2. It looks like you changed the index.html file to link to your
>    personal CSS file. Instead, we'll want to remove that and in the <body>,
>    add a link that directs to your advice page.
>
> Once you do that, you can add, commit, and push and your changes will join
> your pull request here.
>
> xoxo,
> Sharon
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/105-practice/pull/366#issuecomment-485249878>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWBVMEXBLN2ITJSPFFROKDPRRQWZANCNFSM4HHJKVYQ>
> .
>

acodeinprogress commented:
Hey @Zosimi,

Great! I'm going to go ahead and close this pull request so you can focus on your new one.

xoxo,
Sharon
